backend/database/factories/VisitFactory.php

```diff
@@ -7,13 +7,14 @@
 use App\Entities\Session;
 use App\Entities\Visit;
 use App\Entities\Visitor;
+use Carbon\Carbon;
 use Faker\Generator as Faker;
 
 $factory->define(Visit::class, function (Faker $faker) {
     $session = Session::inRandomOrder()->first();
 
     return [
-        'visit_time' => $session->start_session,
+        'visit_time' => $session->start_session->addHours(rand(0, 100))->toDateTimeString(),
```

Comment:
random in factory is supposed to break some test, at least tests that use this factory and do not set visit_time explicitly become unpredictable after these changes. If you need range of visit_time, set it in your test explicitly.